### PR TITLE
Fix header nav styles

### PR DESCRIPTION
**Pull request by vfonic**
Not sure if I updated app version correctly. Should it be minor version update?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "anatine",
   "productName": "Anatine",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (sindresorhus) [inline package.json]: Never bump the version in a PR. It should be up to the maintainer to decide what version to bump, when to bump, and how to bump (I for example, use an automated script) ;)

File: browser.css [hunk 2]
```diff
@@ -19,19 +19,29 @@ html.os-win32 {
...
-	max-width: 800px !important;
 	width: 80% !important;
 }
 #react-root header nav {
 	border-bottom: 1px solid #e1e8ed;
 	background-color: white;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
```
Comment (sindresorhus) [inline browser.css]: Incorrect indentation.

File: browser.css [hunk 3]
```diff
@@ -272,4 +282,3 @@ main > div._3xzA16lK /* profile page */ section > div > div > div > div > div /*
 {
 	display: none;
 }
-
```
Comment (sindresorhus) [inline browser.css:275]: Don't do unrelated changes.

File: browser.css [hunk 4]
```diff
@@ -19,19 +19,29 @@ html.os-win32 {
...
 
 /* navbar */
+#react-root header {
+  position: relative;
+  height: 50px;
+}
+#react-root header nav > div {
+  max-width: 800px !important;
+  margin: 0 auto;
+}
```
Comment (sindresorhus) [inline browser.css:29]: Incorrect indentation.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by vfonic** [comment]
Would it be better if I addressed your comments in a new commit or rebasing like this (and having "outdated diff") is ok?

**Comment by sindresorhus** [comment]
Since the GitHub UI has a squash button now, it's better to add additional commits.

**Comment by sindresorhus** [comment]
Thanks :)
